src/pages/documentation/migrate-to-android.md

```diff
@@ -0,0 +1,89 @@
+
+### Gradle
+
+If you are using Gradle to manage your Adobe Experience Platform Mobile SDK dependencies, the following example shows you how to switch Android 1.x libraries to Android 2.0 libraries in your `build.gradle` file.
+
+```java
+dependencies {
+    //replace sdk-core with Core/Lifecycle/Identity/Signal 2.0
+    //implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+    implementation 'com.adobe.marketing.mobile:core:2.+'
+    implementation 'com.adobe.marketing.mobile:identity:2.+'
+    implementation 'com.adobe.marketing.mobile:lifecycle:2.+'
+    implementation 'com.adobe.marketing.mobile:signal:2.+'
+
+    // replace UserProfile 1.+ with UserProfile 2.0
+    //implementation 'com.adobe.marketing.mobile:userprofile:1.+'
+    implementation 'com.adobe.marketing.mobile:userprofile:2.+'
+
+}
+```
+
+<InlineAlert variant="warning" slots="text"/>
+
+Warning Using dynamic dependency versions is not recommended for production apps. Refer to this [page](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/MobileCore/gradle-dependencies.md) for managing gradle dependencies.
+
+Save the `build.gradle` file and click `Sync Project with Gradle Files` in Android Studio to download the latet Android SDK.
+
+## Update SDK initialization
+
+After you have imported the new Android libraries, you'll need to update SDK initialization code as described below. With Android 2.0, the `MobileCore.start()` API is no longer required. The SDK has simplified initialization and registration of extensions by providing the `MobileCore.registerExtensions()` API. After the given extensions have been registered, the SDK will be initialized and the completion block will be executed. Code which used to reside in the start() block will now reside in the `MobileCore.registerExtensions()`'s completion block.
+
+The following code snippets show the new and correct initialization code required for the Android 2.0 libraries.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
+
+Java
+
+<Tabs query="platform=java"/>
+
+Kotlin
+
+<Tabs query="platform=kotlin"/>
+
+## Replace the deprecated API calls with the new API calls
+
+Several Android 1.x APIs are depreacted in Android 2.0 libraries, you can choose to replace the deprecated API calls in your code with calling the alternative APIs:
+
+| Deprecated API                  | Alternative API                  |
+| :------------------------------ | :------------------------------- |
+| Identity.registerExtension      | MobileCore.registerExtensions    |
+| Signal.registerExtension        |                                  |
+| Lifecycle.registerExtension     |                                  |
+| UserProfile.registerExtension   |                                  |
```

Comment:
Do these not have alternative APIs? I'd assume it's just the MobileCore.registerExtensions, but I want to be doubly sure.

Also, would be good IMO to wrap the APIs in the `` ticks for code lines.

(Also same comment as above for table spacing/whitespace.)

Reply:
Yes, those `registerExtension` APIs should be replaced by the`registerExtensions` API. Ideally, we can merge those empty cells, but I'm not sure if it's supported. Let me know if you have any suggestions.

Reply:
Ahh, I don't believe you can, unfortunately, for markdown tables. As a result, the best thing probably _is_ to be redundant and put it in each of the same tabs.

Reply:
@emdobrin  this PR doesn't contain migration details for all extensions. Can we add those in other PRs?